libs/ngx-dsfr/navigation/src/lib/testing/navigation.harness.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Angular imports
+ */
+import {
+  AsyncFactoryFn,
+  ComponentHarness,
+  TestElement
+} from '@angular/cdk/testing';
+
+/**
+ * This class inherits from ComponentHarness and its purpose
+ * is to serve as a wrapper around an instance of DsfrNavigationComponent
+ * within Unit Tests
+ *
+ * To see more : https://material.angular.io/cdk/test-harnesses/overview
+ */
+export class DsfrNavigationHarness extends ComponentHarness {
+  static hostSelector: string = 'dsfr-navigation';
+
+  private getNavigationElement: AsyncFactoryFn<TestElement> = this.locatorFor('nav');
+  private getAllItemsNavigation: AsyncFactoryFn<TestElement> = this.locatorFor('nav ul li button + div');
```

Comment:
Shouldn't it be called `getFirstNavItem` ?

Reply:
No because it is a multiple DOM selector

Reply:
Nope... 
In that case, it should have been `locatorForAll`.
See https://material.angular.io/cdk/test-harnesses/overview#finding-elements-in-the-components-dom 😉